lib/kdump_utils.pm

```diff
@@ -112,6 +112,16 @@ sub activate_kdump {
     } until (match_has_tag('yast2_console-finished'));
 }
 
+sub activate_kdump_without_yast {
+    # activate kdump by grub, need a reboot to start kdump
+    my $cmd = "if [ -e /etc/default/grub ]; then sed -i '/^GRUB_CMDLINE_LINUX_DEFAULT/ s/\"\$/ crashkernel=256M,high crashkernel=128M,low \"/' /etc/default/grub; fi";
+    script_run($cmd);
+    script_run("cat /etc/default/grub");
+    # sync changes from /etc/default/grub into /boot/grub2/grub.cfg
+    script_run("grub2-mkconfig -o /boot/grub2/grub.cfg");
+    script_run("systemctl enable kdump.service");
```

Comment:
Would you mind to use `assert_script_run` in all cases?

Reply:
OK, `cat /etc/default/grub` just in case some bad thing happened and for information collection, I'd like leave this one in `script_run`. And change next two to use `assert_script_run`.